pyvrp/cpp/Individual_bindings.cpp

```diff
@@ -11,29 +11,20 @@ namespace py = pybind11;
 PYBIND11_MODULE(_Individual, m)
 {
     py::class_<Individual>(m, "Individual")
-        .def(py::init<ProblemData const &,
-                      PenaltyManager const &,
-                      std::vector<std::vector<int>>>(),
+        .def(py::init<ProblemData const &, std::vector<std::vector<int>>>(),
              py::arg("data"),
-             py::arg("penalty_manager"),
-             py::arg("routes"),
-             py::keep_alive<1, 3>())  // keep penalty_manager alive
+             py::arg("routes"))
         .def_property_readonly_static(
             "make_random",                // this is a bit of a workaround for
             [](py::object)                // classmethods, because pybind does
             {                             // not yet support those natively.
                 return py::cpp_function(  // See issue 1693 in the pybind repo.
-                    [](ProblemData const &data,
-                       PenaltyManager const &penaltyManager,
-                       XorShift128 &rng) {
-                        return Individual(data, penaltyManager, rng);
+                    [](ProblemData const &data, XorShift128 &rng) {
+                        return Individual(data, rng);
                     },
                     py::arg("data"),
-                    py::arg("penalty_manager"),
-                    py::arg("routes"),
-                    py::keep_alive<0, 2>());  // keep penalty_manager alive
+                    py::arg("routes"));
```

Comment:
```suggestion
                    py::arg("rng"));
```

I'm reasonably sure this was wrong before.